### PR TITLE
PCHR-1273: Calendar Styles

### PR DESCRIPTION
**Pull request by deb1990**
**Changes**

1. Calendar styles moved from HTML to CSS files. The styles were part of the HTML as the plan was to generate Classes dynamically using angular. But it is not the case any more, also the moved classes were meant to be static.

2. Styles for multi select has been added.

### CODE REVIEW

File: civihr_default_theme/scss/components/_ui-select.scss [hunk 1]
```diff
@@ -53,6 +53,25 @@
...
+    font-family: "FontAwesome";
+    display: inline-block;
+    height: 33px;
+    line-height: 35px;
+    position: absolute;
+    right: 0;
+    text-align: center;
+    width: 32px;
+    z-index: 1;
+    -webkit-font-smoothing: antialiased;
```
Comment (dhavalsharma) [inline civihr_default_theme/scss/components/_ui-select.scss]: font smoothing has limited support [across browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth)? Also only webkit is used here?

**Review by dhavalsharma — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/components/_ui-select.scss (hunk 1 above)
Comment (deb1990) [reply civihr_default_theme/scss/components/_ui-select.scss]: I used `font-smoothing` code from `_custom-select.scss` which was already present, now moved the code to a mixin and added `moz` prefix.
Reference: https://davidwalsh.name/font-smoothing

File: civihr_default_theme/scss/components/_leave-calendar.scss [hunk 2]
```diff
@@ -107,7 +107,26 @@ $chr-leave-calendar-legend-size:                  20px;
...
+  border-color: #8ec68a;
+}
+
+.chr_leave-calendar__item--type--weekend {
+  background-color: #c7cbd0 !important;
+  border-color: #c7cbd0 !important;
+}
+
+.chr_leave-calendar__item--type--non-working {
+  background-color: #41afcb !important;
```
Comment (AkA84) [inline civihr_default_theme/scss/components/_leave-calendar.scss:125]: Why some have `important!` and some do not?

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/components/_leave-calendar.scss (hunk 2 above)
Comment (deb1990) [reply civihr_default_theme/scss/components/_leave-calendar.scss:125]: Kept `!important` for `weekend` and `non-working`day to give them more priority. 
Example: If a date is weekend and also a leaveRequest is there for that date, showing colors for the `weekend` as leave is not calculated for that day.

**Review by AkA84 — APPROVED** [review]

**Review by dhavalsharma — APPROVED** [review]

**Pull request merged**